src/time.rs

```diff
@@ -488,3 +523,22 @@ impl PureTime {
         self.hour as u32 * 3_600 + self.minute as u32 * 60 + self.second as u32
     }
 }
+
+#[derive(Debug, Clone)]
+pub enum SecondsPrecisionOverflowBehavior {
+    Truncate,
+    Error,
+}
```

Comment:
might be good to implement `TryFrom<String>` for this here.

Reply:
done